subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/daemon/JavaCompilerDaemonReuseIntegrationTest.groovy

```diff
@@ -44,6 +44,41 @@ class JavaCompilerDaemonReuseIntegrationTest extends AbstractCompilerDaemonReuse
         return new TestJavaComponent()
     }
 
+    def "compiler daemon reuse can be disabled"() {
```

Comment:
Should we mention how to disable behaviour in case of troubles in the release notes?

Reply:
No, this isn't going to be a public option

Reply:
I wonder if it should be, just in case 🤔